bindings_node/src/groups.rs

```diff
@@ -119,7 +121,7 @@ impl NapiGroup {
       .send_message_optimistic(encoded_content.encode_to_vec().as_slice())
       .map_err(ErrorWrapper::from)?;
 
-    Ok(id)
+    Ok(hex::encode(id.clone()))
```

Comment:
maybe not important, but keep in mind that `hex::encode` will return a string without `0x`, it could trip some apps that expect that. `format!("0x{}", hex::encode(id.clone())` could add the `0x`